README.md

```diff
@@ -86,17 +84,11 @@ In the quality profile, activate some rules from Checkstyle and run an analysis
 [teamcity]:https://teamcity.jetbrains.com/viewType.html?buildTypeId=Checkstyle_SonarCheckstyleIdeaInspectionsMaster
 [teamcity img]:https://teamcity.jetbrains.com/app/rest/builds/buildType:(id:Checkstyle_SonarCheckstyleIdeaInspectionsMaster)/statusIcon
 
-[sonar]:https://sonarcloud.io/dashboard?id=com.github.checkstyle%3Acheckstyle-sonar-plugin-parent
-[sonar img]:https://sonarcloud.io/api/project_badges/measure?project=com.github.checkstyle%3Acheckstyle-sonar-plugin-parent&metric=alert_status
```

Comment:
I don't see a duplicate `[sonar]` badge like the others. Should this have just been changed from `checkstyle-sonar-plugin-parent` to `checkstyle-sonar-plugin`?

Reply:
Thank you! :) Yes, after an analysis it should become `checkstyle-sonar-plugin` since the `artifactId` changed in our `pom.xml`. Is there a possibility to rename the current SonarCloud project key since the old one will not exist anymore but analysis will create a new project (key: `com.github.checkstyle:checkstyle-sonar-plugin`)?